apps/api/src/music/music.controller.ts

```diff
@@ -14,14 +14,16 @@ import {
 import { diskStorage } from 'multer';
 import { MusicService } from './music.service';
 import { ApiOperation, ApiTags } from '@nestjs/swagger';
-import {
-  multerDiskStorageDestination,
-  audioFileFilter,
-  multerDiskStorageFilename,
-} from '../utils/mutler';
 import { FileInterceptor } from '@nestjs/platform-express';
 import { CreateMusicDto } from './dto/create-music.dto';
 import { UpdateMusicDto } from './dto/update-music.dto';
+import {
+  imageFileFilter,
+  multerDiskStorageMusicAudioFileDestination,
+  multerDiskStorageFilename,
+  multerDiskStorageMusicCoverFileImageDestination,
+  audioFileFilter,
+} from '@musica/core';
 
 @ApiTags('Music')
 @Controller('music')
```

Comment:
It would be nice if we use a plural name insead.

```suggestion
@Controller('songs')
```

Reply:
This has a cascading effect on the entire naming of project folder. Only adding '-s' can be an acceptable suggestion. Moreover, music is as general as song.